misc/python/materialize/cli/ci_closed_issues_detect.py

```diff
@@ -45,6 +45,8 @@
     | @pytest.mark.skip
     # Used in output-consistency framework
     | YesIgnore
+    # Used in proto files
+    // buf breaking: ignore
```

Comment:
This is a verbose regex, so spaces have to be escaped
```suggestion
    //\ buf\ breaking:\ ignore
```